examples/scala/src/main/scala/ClusterExample.scala

```diff
@@ -61,15 +61,13 @@ object ClusterExample {
 
             // Send INFO REPLICATION with routing option to all nodes
             infoReplResps <- client.info(
-                InfoOptions.builder()
-                    .section(InfoOptions.Section.REPLICATION)
-                    .build(),
+                Array(Section.REPLICATION),
                 ALL_NODES
             ).asScala
             _ = Logger.log(
                 Logger.Level.INFO,
                 "app",
-                "INFO REPLICATION responses from all nodes are=\n$infoReplResps",
+                s"INFO REPLICATION responses from all nodes are\n${infoReplResps.getMultiValue()}",
```

Comment:
```suggestion
                s"INFO REPLICATION responses from all nodes are:\n${infoReplResps.getMultiValue()}",
```
nit: might add a bit of readability